src/compass/utils/age.py

```diff
@@ -17,7 +17,7 @@
 
 def run(cslc_file, cr_file, csv_output_file=None, plot_age=False,
         correct_set=False, mission_id='S1', pol='VV', ovs_factor=128,
-        margin=32):
+        margin=32, carrier=True, unflatten=True):
```

Comment:
```suggestion
        margin=32, apply_az_ramp=True, unflatten=True):
```
nit - rename for clarity and consistency with `run` and `main`.